packages/opentelemetry-plugin-mongodb/package.json

```diff
@@ -39,10 +39,13 @@
   },
   "devDependencies": {
     "@types/mocha": "^5.2.7",
-    "@types/node": "^12.6.9",
+    "@types/mongodb": "^3.2.3",
+    "@types/node": "^12.7.2",
+    "@types/shimmer": "^1.0.1",
     "codecov": "^3.5.0",
     "gts": "^1.1.0",
     "mocha": "^6.2.0",
+    "mongodb": "^3.3.0",
     "nyc": "^14.1.1",
     "rimraf": "^3.0.0",
     "tslint-microsoft-contrib": "^6.2.0",
```

Comment:
missing `"tslint-consistent-codestyle": "^1.16.0",`

see other plugin for integration with tslint

```json
{
  "rulesDirectory": ["node_modules/tslint-microsoft-contrib"],
  "extends": ["../../tslint.base.js", "./node_modules/tslint-consistent-codestyle"]
}
```